README.md

```diff
@@ -51,11 +51,13 @@ The following environment variables needs to be defined:
   * `OS_AUTH_URL`
   * `OS_PASSWORD`
   * `OS_USERNAME`
+  * `OS_DOMAIN_NAME`
 
-Author
+Authors
 ------
 
   * François Deppierraz <francois.deppierraz@infraly.ch>
+  * Saverio Proto <saverio.proto@switch.ch>
```

Comment:
Already added in https://github.com/infraly/k8s-on-openstack/commit/241d9671baff25ef3a2cb4ba5984586faa89b2fc